packages/client/src/api/api.service.ts

```diff
@@ -53,8 +53,15 @@ export class ApiService {
    * @deprecated The method should not be used - Use markMessageAs instead.
    */
   async markMessageAsSeen(messageId: string | string[]): Promise<any> {
+    const messageIdString =
+      !messageId || typeof messageId === 'object'
+        ? Array.isArray(messageId)
+          ? messageId.toString()
+          : ''
+        : messageId.toString();
```

Comment:
```suggestion
    const messageIdString = messageId ? messageId.toString() : '';
```

Perhaps can be simplified here aswell.